app/code/Magento/Bundle/Test/Unit/Model/LinkManagementTest.php

```diff
@@ -533,12 +542,15 @@ public function testAddChildCouldNotSave()
         $this->expectException(CouldNotSaveException::class);
 
         $productLink = $this->getMockBuilder(LinkInterface::class)
-            ->setMethods(['getSku', 'getOptionId', 'getSelectionId'])
+            ->setMethods(['getSku', 'getOptionId', 'getSelectionId', 'getWebSiteId'])
```

Comment:
This method doesn't exist anymore on the Link, please remove it.

Reply:
Done